### ISSUE TITLE
Rework login screen

### ISSUE THREAD

**Issue by moufmouf** [thecodingmachine/workadventure#128]
Proposal: make a login screen in 2 scenes.
- First scene: Big logo (designed by @soy) and enter your name
- Second scene: choose your character

Choosing your character should be possible using the keyboard only.

**Comment by moufmouf** [comment]
Almost completely done in #129.
Only the logo is lacking to close this issue.
